src/pages/docs/[...slug].js

```diff
@@ -88,6 +89,16 @@ export default function Docs(props) {
   let toc = props.toc;
   // get meta
   let meta = extractMetaFromMarkdown(props.markdown);
+
+  // getting the title and description
+  const titleRegexMeta = /(?<=title: ).+?(\n)/;
+  const titleRegexMd = /(?<=# ).+/;
+  const titleMeta = meta?.match(titleRegexMeta);
+  const titleMd = props.markdown.match(titleRegexMd);
+  const title = (titleMeta || titleMd)[0];
+  const descRegex = /(?<=description: >-).+/is;
+  const description = meta.match(descRegex)[0];
+
```

Comment:
generic zesty share image https://github.com/zesty-io/nextjs-website/blob/d32f0619198b793c61204dce49bec244798be17e/src/components/ZestyHead.js#L7 this should be the default image, otherwise it should be the first image found in the document.